### ISSUE TITLE
Create a "Server Install" section specifically for installing IPFS Cluster and not Go-IPFS.

### ISSUE THREAD

**Issue by johnnymatthews** [ipfs/ipfs-docs#355]
As discussed in #253, users wanting to install IPFS in a server environment should install IPFS Cluster. This section should:

- [x] Explain what IPFS Cluster is.
- [x] Explain why server environments benefit from IPFS Cluster.
- [x] Walk the user through installing IPFS Cluster via:
  - [x] Docker
  - [ ] ~~Ansible~~
- [x] Link to further resources that server environment would benefit from.

**Comment by lizelive** [comment]
Im hyped about these docs. The current ones are a bit hard to follow.

**Comment by johnnymatthews** [comment]
@lizelive I'm glad you're excited! PR #467 is currently in review that you could take a look at. Right now it's just a simple quick start for setting up a local IPFS Cluster environment using Docker. We'd really value your feedback!
